MapboxNavigation/NavigationMapView.swift

```diff
@@ -672,7 +672,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
                 arrowSymbolLayerCasing.minimumZoomLevel = minimumZoomLevel
                 arrowSymbolLayerCasing.iconImageName = NSExpression(forConstantValue: "triangle-tip-navigation")
                 arrowSymbolLayerCasing.iconColor = NSExpression(forConstantValue: UIColor.defaultArrowStroke)
-                arrowSymbolLayerCasing.iconRotationAlignment = NSExpression(forConstantValue: NSValue(mglIconRotationAlignment: .map))
+                arrowSymbolLayerCasing.iconRotationAlignment = arrowSymbolLayer.iconRotationAlignment
```

Comment:
The idea behind https://github.com/mapbox/mapbox-navigation-ios/pull/1076#discussion_r178962197 was that we’d do the same thing for the dozens of other properties in this file.